yolov8_ros/yolov8_ros/yolov8_ros/yolov8_node.py

```diff
@@ -278,9 +328,12 @@ def image_cb(self, msg: Image) -> None:
 
                 detections_msg.detections.append(aux_msg)
 
+                iceberg_detections_msg.icebergdetections.append(iceberg_aux_msg)
+
             # publish detections
             detections_msg.header = msg.header
             self._pub.publish(detections_msg)
```

Comment:
What is the difference between iceberg detections and the other detections?

Reply:
Removed other detections publisher. Kept iceberg detections publisher.